app/controllers/api/v0x1.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V0x1
     class RootController < ApplicationController
       def openapi
-        render :json => Api::Docs["0.1"].to_json
+        render :json => Api::Docs["0.1"].content.to_json
```

Comment:
This change is not clear to me.  Why doesn't the to_json itself just call content?

Reply:
it does, it's just that the spec should not include the content key which is what to_json serializes it as:

```
{
  "content" : {
    "openapi" : "3.0.0",
    "info" : { ... },
    "security" : { ... },
    "paths" : { 
       ...
    }
    ...
}
```

should just be:

```
{
  "openapi" : "3.0.0",
  "info" : { ... },
  "security" : { ... },
  "paths" : {
     ...
  }
  ...
}
```

same issue with the current v0.1.0 swagger flavor.